### PR TITLE
fix(proxy): propagate x-github-token

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

File: test/content-proxy.test.js [hunk 1]
```diff
@@ -30,15 +30,22 @@ describe('Content Proxy Tests', () => {
   before(async () => {
     nock('https://adobeioruntime.net')
       .get((uri) => uri.startsWith('/api/v1/web/helix'))
-      .reply((uri) => {
+      .reply(async function cb(uri) {
```
Comment (dominique-pfister) [inline test/content-proxy.test.js:33]: hm, isn't that the same as:
```
.reply(async (uri) => {
  ...
}
```
don't we usually prefer the arrow notation?

Comment (tripodsan) [reply test/content-proxy.test.js:33]: yes. but the request headers are only available via `this.req.headers`

Comment (dominique-pfister) [reply test/content-proxy.test.js:33]: haha, right, I forgot about the `this` in an arrow function 😉

**Review by dominique-pfister — APPROVED** [review]

**Pull request merged**